scripts/prep-release.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Helper script to prepare a release for the Go SDK.
```

Comment:
Let's make a check here that the working directory is clean (i.e. there are no uncommitted changes). This will save us someday from releasing some obscure WIP code

Reply:
Added!